pkg/controller/operatingsystemconfig/os-common/example/operatingsystemconfig.yaml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
remove, same reason as above, entire example folder should go away

Reply:
I don't understand the logic of removing anything that makes the library useful and reusable: README, examples, ...

Reply:
Logic: Double maintenance effort. Let's only maintain examples for concrete controllers, not for any libraries (examples would anyway not work without a concrete controller implementation).